dotnet/src/Connectors/Connectors.AzureOpenAI/Connectors.AzureOpenAI.csproj

```diff
@@ -21,10 +21,18 @@
     <Description>Semantic Kernel connectors for Azure OpenAI. Contains clients for text generation, chat completion, embedding and DALL-E text to image.</Description>
   </PropertyGroup>
 
+  <ItemGroup>
```

Comment:
Just curious: Why do we need to exclude this and the one below from compilation?